src/pages/Preview/Preview.tsx

```diff
@@ -0,0 +1,369 @@
+          </div>
+          <Button
+            variant="contained"
+            onClick={() => (window.location.href = "/payroll")}
+          >
+            <ArrowLeft />
+            Go back to dashboard
+          </Button>
+        </Box>
+      )}
+
+      {!isButtonClicked && (
+        <Box
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "space-between",
+          }}
+        >
+          <LocalizationProvider dateAdapter={AdapterDayjs}>
+            <DemoContainer components={["DatePayroll"]}>
+              <span>Select the date range:</span>
+              <span>From</span>
+              <DatePicker
+                // label="Start Date"
+                value={startDate}
+                onChange={(value) => {
+                  if (value) {
+                    setStartDate(value);
+                  }
+                }}
+                readOnly
+              />
+              <span>to</span>
+              <DatePicker
+                // label="End Date"
+                value={endDate}
+                onChange={(value) => {
+                  if (value) {
+                    setEndDate(value);
+                  }
+                }}
+              />
+            </DemoContainer>
+          </LocalizationProvider>
+
+          <Button variant="contained" color="primary" onClick={handleClickOpen}>
+            Run Payroll
+            <CaretRight size={25} />
+          </Button>
+
+          <Dialog
+            open={open}
+            onClose={handleClose}
+            aria-labelledby="alert-dialog-title"
+            aria-describedby="alert-dialog-description"
+          >
+            <DialogTitle id="alert-dialog-title">{"ICON"}</DialogTitle>
+            <DialogContent>
+              <DialogContentText id="alert-dialog-description">
+                Ready to run the payroll?
+              </DialogContentText>
+            </DialogContent>
+            <Typography>
+              {uniqueSeasonName}
+            </Typography>
+            <Typography>
+              {!!startDate && formatDate(startDate)}{" - "}{formatDate(endDate)}
+            </Typography>
+            <Typography>
+              {netPay}
+            </Typography>
+            <DialogActions>
+              <Button onClick={handleClose} color="primary">
+                Cancel
+              </Button>
+              <Button onClick={handleConfirm} color="primary" autoFocus>
+                Confirm
+              </Button>
+            </DialogActions>
+          </Dialog>
+        </Box>
+      )}
+
+      <Box display="flex" flexDirection="column" flexGrow={1} pb={3}>
+        <Grid container spacing={2}>
+          <Grid item xs={3}>
+            <Typography variant="h6" gutterBottom>
+              PAY PERIOD
+            </Typography>
+          </Grid>
+          <Grid item xs={3}>
+            <Typography variant="h6" gutterBottom>
+              TOTAL NET PAY
```

Comment:
uppercase should be on the design, we should not use Typography, but in this case Label from our custom components